mitosheet/src/jupyter/jupyterUtils.tsx

```diff
@@ -52,16 +52,17 @@ export const overwriteAnalysisToReplayToMitosheetCall = (oldAnalysisName: string
 }
 
 
-export const writeGeneratedCodeToCell = (analysisName: string, code: string[], telemetryEnabled: boolean, publicInterfaceVersion: PublicInterfaceVersion): void => {
+export const writeGeneratedCodeToCell = (analysisName: string, code: string[], telemetryEnabled: boolean, publicInterfaceVersion: PublicInterfaceVersion, oldCode?: string[]): void => {
```

Comment:
Ok -- so it's optional because upgraded analyses might not have it?

Reply:
Yes

Reply:
I went ahead and removed that because we're using `analysisData`, which doesn't list `code` as optional